mqtt/.dockerignore

```diff
@@ -4,7 +4,6 @@
 .gitignore
 .vscode
 state
-target
```

Comment:
We should confirm removing this has no side effects. I'm assuming it won't?

Reply:
`target` contains a huge amount of binary artifacts that will be copied into the builder image context. But we don't do that anymore, right?

Reply:
Is it possible to copy build output (executable) at some point in some other folder? I would rather want to keep "target" in the .dockerignore

Reply:
This should be reverted, right?